### PR TITLE
readme: polishing readme statements

### PR DESCRIPTION
**Pull request by legendecas**
##### Checklist

- [x] documentation is changed or added

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # ShadowNode
 
-The Node.js Runtime in shadow, which supports NPM packages partially for embeddable platform.
+The Node.js runtime in shadow, enables N-API and vast Node.js packages on embedded devices.
```
Comment (yorkie) [inline README.md]: "The Node.js runtime in device"?

File: README.md [hunk 2]
```diff
@@ -12,55 +12,54 @@ Currently supports the following platform:
 - [x] OSX
 - [x] Linux
 
+The project is another runtime for your [Node.js][] packages, while ShadowNode is designed to be used on memory limited devices. It's inspired and forked from the awesome project [Samsung/iotjs][].
```
Comment (yorkie) [inline README.md]: Should "packages" be corrected to "applications"?

Comment (yorkie) [inline README.md]:
```suggestion
The project is another runtime for your [Node.js][] packages, while it's designed to be used on memory limited devices. It's inspired and forked from the awesome project [Samsung/iotjs][].
```

File: README.md [hunk 3]
```diff
@@ -12,55 +12,54 @@ Currently supports the following platform:
...
+
 ## Quick Start
 
-This project is another runtime for your [Node.js][] app, which is to be used in low memory devices. It's inspired and forked
-from the original awesome project [Samsung/iotjs][].
+To get started with ShadowNode, you could download prebuilt Linux/macOS binaries on [Release Page](https://github.com/Rokid/ShadowNode/releases).
```
Comment (yorkie) [inline README.md]:
```
To get started with ShadowNode, you could download prebuilt binaries on [Release Page](https://github.com/Rokid/ShadowNode/releases) for the following targets:

- Linux x86
- MacOS x86
```

File: README.md [hunk 4]
```diff
@@ -12,55 +12,54 @@ Currently supports the following platform:
...
 ```
 
-For Additional information see [Getting Started](docs/Getting-Started.md).
+For additional information see [Getting Started](docs/Getting-Started.md).
 
-## What's different with Node.js
+## Compared with Node.js
+
+ShadowNode is not designed to be ran identical code that ran on Node.js.
+While embedded environments are experiencing limited resources on runtime, the packages to be ran on ShadowNode shall be rewritten in a resource compact way. However for the very initial thought of sharing the Node.js vast module ecosystem, we would like make ShadowNode compatible with Node.js with our efforts.
```
Comment (yorkie) [inline README.md]:
```suggestion
While embedded environments are experiencing limited resources on runtime, the applications to be ran on ShadowNode shall be rewritten in a resource compact way. However for the very initial thought of sharing the Node.js vast module ecosystem, we would like make ShadowNode compatible with Node.js with our efforts.
```

File: README.md [hunk 5]
```diff
@@ -81,18 +80,18 @@ as possible, and the following are the status:
 - [UDP/Datagram](docs/api/DGRAM.md)
 - [Zlib](docs/api/Zlib.md)
 
-In desktop and embedable ecosystem, the service via `DBus` is quite normal, in shadow-node, the `DBus`
-will be the built-in module. For detailed API, see [DBUS API](docs/api/DBUS.md).
+In desktop and embeddable ecosystem, the service via `DBus` is quite normal, in ShadowNode, the `DBus`
+will be a built-in module. For detailed API, see [DBus API](docs/api/DBUS.md).
```
Comment (yorkie) [inline README.md]: The DBus have a plan to be deprecated, so I think we could remove this description from README :)

**Comment by yorkie** [comment]
By the way, I think we could emphasize the works on N-API and standards, too.

**Comment by yorkie** [comment]
Ping @legendecas, any updates?

File: README.md (hunk 2 above)
Comment (legendecas) [reply README.md]: Statement "package" might including "application" as well, whilst "application" limits the statement to a narrower definition of programs could be ran.

File: README.md (hunk 1 above)
Comment (legendecas) [reply README.md]: The "Node.js packages" does include  "packages from NPM", but not vice versa since not all packages are published to NPM.

**Pull request merged**